pallets/staking/src/lib.rs

```diff
@@ -272,10 +278,12 @@ pub mod pallet {
 
             ensure!(!Self::is_validator(&acc), <Error<T>>::ValidatorExists);
 
-            ensure!(
-                bond >= <StakingMinValidatorBond<T>>::get(),
-                <Error<T>>::ValidatorBondBelowMin
-            );
+            if !Self::is_invulnerable(&acc) {
+                ensure!(
+                    bond >= <StakingMinValidatorBond<T>>::get(),
+                    <Error<T>>::ValidatorBondBelowMin
+                );
+            }
```

Comment:
@rajesh-nodle it looks like this PR contains both changes to the staking pallet and code to add it to eden. Shouldn't we break into two separate PRs?